de.z_ray.Facetracker.json

```diff
@@ -0,0 +1,44 @@
+{
+  "app-id": "de.z_ray.Facetracker",
```

Comment:
Can you put a link to the github or a link to this software somewhere on https://z-ray.de/ I can't find any info to associate the two.

Reply:
Certainly! I added a simple HTML file to the root of that webpage linking back to this PR: https://z-ray.de/flathub.html